sdk/spring-sdk/src/test/java/kalix/springsdk/badwiring/action/IllDefinedAction.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.springsdk.badwiring.action;
+
+import kalix.javasdk.action.Action;
+import org.springframework.stereotype.Component;
+
+@Component
+public class IllDefinedAction extends Action {}
```

Comment:
what makes a Kalix component ill-defined is the mix with Spring stereotypes. That make them available for wiring in the Spring context and that's exactly what we block with the `BeanPostProcessor`